pkgs/development/libraries/opentracing-cpp/default.nix

```diff
@@ -15,7 +21,6 @@ stdenv.mkDerivation rec {
     description = "C++ implementation of the OpenTracing API";
     homepage = "https://opentracing.io";
     license = lib.licenses.asl20;
-    maintainers = with lib.maintainers; [ rob ];
+    maintainers = [ ];
```

Comment:
This package should  be deleted.

Reply:
It is a dependency of a nginx module so we cannot just delete it.